pages/people.md

```diff
@@ -62,6 +60,14 @@ $(function() {
 | vikrams       | Vikram Srivastava                                 | [Cloudera][CLOUDERA]                   | [PT][PT] |
 | vines         | John Vines                                        | [sqrrl][SQRRL]                         | [ET][ET] |
 
+## Emeritus PMC and Committers
```

Comment:
I'm worried that this implies "Emeritus PMC" and "Emeritus Committers", when it's really "Emeritus PMC, but still current Committers". I can't think of a good way to communicate that succinctly in a section header. I thought about reversing the order, as in "Committers and Emeritus PMC" or "Committers Only (PMC Emeritus)", but then what if we have somebody who accepts an invite as committer, but declines being PMC. Then, they are "Committer only, never PMC". Will we want to have a separate section for that case? Or do we want to include them in this same section with an appropriate section header for both situations (and then mark inside the table that they are Emeritus for PMC).

I'm not sure which direction is best... but I'd prefer not to imply that they are Emeritus Committers when their Committer privileges are still held, and only the PMC status is Emeritus.

Reply:
> I'm worried that this implies "Emeritus PMC" and "Emeritus Committers"

I agree, tt can definitely read that way.

> I can't think of a good way to communicate that succinctly

Looking at what you suggested, I like "Committers Only (PMC Emeritus)" and its only a few bytes longer so it still succinct. 

> but then what if we have somebody who accepts an invite as committer, but declines being PMC

We can worry about that case in the future when it happens.  We could create another section or maybe add a column to the table.

Reply:
I pushed an update to the section name in 17591e0595bf823896807dd6c9fd41291a035833